### PR TITLE
[release-2.4] Updated file formatting with prettier

### PR DESCRIPTION
**Pull request by dislbenn**
PR for fixing the linting job for the PR: https://github.com/stolostron/search-e2e-test/pull/176.

### CODE REVIEW

File: download-clis.sh [hunk 1]
```diff
@@ -13,4 +13,7 @@ chmod 755 ./clis-unpacked/kubectl
 mv ./clis-unpacked/oc /usr/local/bin/oc
 mv ./clis-unpacked/kubectl /usr/local/bin/kubectl
 
+# Install htpasswd utility
+apt-get update && apt-get install -y apache2-utils
+
 echo 'oc and kubectl cli install completed.'
```
Comment (jlpadilla) [inline download-clis.sh:19]: Minor suggestion to help us debug problems later.
- Rename the file to something more generic like `install-dependencies.sh`
- Update the echo message to include htpasswd.

File: build/rbac-setup.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@
 # DIR="$( find $HOME -type d -name "search-e2e-test" )"
 # using abosolute path, relative path is not working in travis
 
-set -e
+# set -e
```
Comment (jlpadilla) [inline build/rbac-setup.sh:14]: I'm ok with disabling this while we integrate with prow, but let's add an `echo` to log a message to remind us to revert this change.
```suggestion
# set -e
echo "WORKAROUND: Temporarily allowing `rbac-setup.sh` to continue with errors while we setup the prow build."
```

File: start-tests.sh [hunk 3]
```diff
@@ -364,15 +364,15 @@ fi
 
 if [[ "$SKIP_UI_TEST" == false ]]; then
   # Displaying cypress environment variables, so we know all of the ones that are being passed successfully.
-  env | grep "cypress_" -i
+  # env | grep "cypress_" -i
```
Comment (jlpadilla) [inline start-tests.sh:367]: Please log messages to reming us to revert the 4 changes in this file.  See comment above for example.

File: build/rbac-setup.sh (hunk 2 above)
Comment (dislbenn) [reply build/rbac-setup.sh]: Re-added this line. This should be fixed.

File: download-clis.sh (hunk 1 above)
Comment (dislbenn) [reply download-clis.sh]: Renamed the file and updated the echo messages.

File: start-tests.sh (hunk 3 above)
Comment (dislbenn) [reply start-tests.sh]: Updated this command to remove password env variables.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold
